backend/src/especialidades/entities/especialidade.entity.ts

```diff
@@ -1,6 +1,7 @@
-import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import { Entity, Column, PrimaryGeneratedColumn, Unique } from 'typeorm';
 
-@Entity()
+@Entity('especialidades')
+@Unique(['nome']) 
 export class Especialidade {
   @PrimaryGeneratedColumn()
   id: number;
```

Comment:
precisa manter o ```@Entity()``` pq se não quebra as migrations.

```suggestion
@Entity()
export class Especialidade {
  @PrimaryGeneratedColumn()
  id: number;
  
  @Column({ unique: true })
  name: string;
```